### PR TITLE
add resnet50 model test

### PR DESCRIPTION
**Pull request by Ldpe2G**

### CODE REVIEW

File: oneflow/python/test/modules/resnet50_model.py [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+    def __init__(
+        self,
+        block: Type[Union[BasicBlock, Bottleneck]],
+        layers: List[int],
+        num_classes: int = 1000,
+        zero_init_residual: bool = False,
+        groups: int = 1,
+        width_per_group: int = 64,
+        replace_stride_with_dilation: Optional[List[bool]] = None,
+        norm_layer: Optional[Callable[..., nn.Module]] = None,
```
Comment (daquexian) [inline oneflow/python/test/modules/resnet50_model.py:154]: 这些参数也应该被测试覆盖

File: oneflow/python/test/modules/resnet50_model.py [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+        super(BasicBlock, self).__init__()
+        if norm_layer is None:
+            norm_layer = nn.BatchNorm2d
+        if groups != 1 or base_width != 64:
+            raise ValueError("BasicBlock only supports groups=1 and base_width=64")
+        if dilation > 1:
+            raise NotImplementedError("Dilation > 1 not supported in BasicBlock")
+        # Both self.conv1 and self.downsample layers downsample the input when stride != 1
+        self.conv1 = conv3x3(inplanes, planes, stride)
+        self.bn1 = norm_layer(planes)
```
Comment (wyg1997) [inline oneflow/python/test/modules/resnet50_model.py:106]: 上面norm_layer的类型注解是Optional而且默认是None，这里直接判断是否是None，是的话就跳过是不是更好，这样也就不需要FakeBn了吧？

Comment (Ldpe2G) [reply oneflow/python/test/modules/resnet50_model.py:106]: 他原来实现的逻辑是，如果用户没有设置，norm_layer 是none的话，默认就是那么在init哪里会把 Norm_layer设置成 batchnorm，然后下面代码逻辑都是假设norm_layer是有设置的情况。所以我才偷懒，搞了个fakebn

Comment (wyg1997) [reply oneflow/python/test/modules/resnet50_model.py:106]: 嗯嗯，那暂时先这么写好了

**Review by wyg1997 — APPROVED** [review]
